targetFunctions/+domainsTF/+customLayers/processCustomFunction.m

```diff
@@ -36,11 +36,11 @@
         coder.varsize('thisContrastLayers2',[10000, 6],[1 1]);
 
         if isnan(str2double(functionHandle))
-            [thisContrastLayers1, allRoughs(i)] = callMatlabFunction(params,i,functionHandle,thisBulkIn,thisBulkOut,numberOfContrasts,1);
-            [thisContrastLayers2, ~] = callMatlabFunction(params,i,functionHandle,thisBulkIn,thisBulkOut,numberOfContrasts,2);
+            [thisContrastLayers1, allRoughs(i)] = callMatlabFunction(functionHandle, params, thisBulkIn, bulkOut, i, 1);
+            [thisContrastLayers2, ~] = callMatlabFunction(functionHandle, params, thisBulkIn, bulkOut, i, 2);
```

Comment:
Why change from "thisBulkOut" to "bulkOut" but not "thisBulkIn" to "bulkIn"?

Reply:
Since both are changed to vectors above.

Reply:
`thisBulkOut` contains only a single value i.e `bulkOut[contrast]` but according to Arwel the custom function should receive the full array to maintain backward compatibility with RasCAL

Reply:
Makes sense, so why not do the same with bulk in?

Reply:
My guess is that the bulkIn was not passed like that in RasCAL. Is this correct @arwelHughes ?